src/inspect_ai/model/_call_tools.py

```diff
@@ -134,6 +135,7 @@ async def call_tool_task(
                         )
                     # unwrap exception group
                     except Exception as ex:
+                        # TODO: This appears to "lose" the LimitExceededError.conversation
```

Comment:
Part of the bigger discussion on whether we actually need this for the non-solver cases.